sources/operator.2.6.0.pg-operator.json

```diff
@@ -0,0 +1,252 @@
+            "status": "available"
+          },
+          "15.12": {
+            "critical": false,
+            "image_hash": "9d33160904b7862d03c1018e4bf80247ea175918b7b0b4d4907e175045ddf4d1",
+            "image_path": "percona/percona-postgresql-operator:2.6.0-ppg15.12-pgbackrest2.54.2",
+            "status": "recommended"
+          },
+          "16.4": {
+            "critical": false,
+            "image_hash": "d636aa73c6f74075d8c2b9bbf51892db94e8d698e66d23ce2b1f612df5a0ddde",
+            "image_path": "percona/percona-postgresql-operator:2.5.0-ppg16.4-pgbackrest2.53-1",
+            "status": "available"
+          },
+          "16.8": {
+            "critical": false,
+            "image_hash": "eca4f0153fd75c87bb35e54e5358da458a502bcc7671a798ddf60f6a87246ba8",
+            "image_path": "percona/percona-postgresql-operator:2.6.0-ppg16.8-pgbackrest2.54.2",
+            "status": "recommended"
+          },
+          "17.2": {
+            "critical": false,
+            "image_hash": "a3641d58a49fe4f771f3638c9fa18c71dd2f9aba1054e5693d3756134676cb3e",
+            "image_path": "percona/percona-postgresql-operator:2.6.0-ppg17.2-pgbackrest2.54.0",
+            "status": "available"
+          }
+        },
+        "pgbouncer": {
+          "13.16": {
+            "critical": false,
+            "image_hash": "e1692b9b9ec02ee232449e5da6df464708673b637c66b109db4c21836d5cae8f",
+            "image_path": "percona/percona-postgresql-operator:2.5.0-ppg13.16-pgbouncer1.23.1",
+            "status": "available"
+          },
+          "13.20": {
+            "critical": false,
+            "image_hash": "a1c25e9834fbc8ad58477fc47ef868ccf3c696bd488114efc8c7f61c66961356",
+            "image_path": "percona/percona-postgresql-operator:2.6.0-ppg13.20-pgbouncer1.24.0",
+            "status": "recommended"
+          },
+          "14.13": {
+            "critical": false,
+            "image_hash": "1652e89ad49ab50242b683b5107d70d28b6f1b1cadc3566411dc58902c146aa2",
+            "image_path": "percona/percona-postgresql-operator:2.5.0-ppg14.13-pgbouncer1.23.1",
+            "status": "available"
+          },
+          "14.17": {
+            "critical": false,
+            "image_hash": "6ffc19f626b738b096635a0b1a2e4fbb28f800723759c58bd8491e9857b2fc19",
+            "image_path": "percona/percona-postgresql-operator:2.6.0-ppg14.17-pgbouncer1.24.0",
+            "status": "recommended"
+          },
+          "15.8": {
+            "critical": false,
+            "image_hash": "c048dbd2cdf86c759d2a5e44028455c4a7b7f951892e390b41491769034b99b7",
+            "image_path": "percona/percona-postgresql-operator:2.5.0-ppg15.8-pgbouncer1.23.1",
+            "status": "available"
+          },
+          "15.12": {
+            "critical": false,
+            "image_hash": "9ef6204ebf626ee85d2a2afd405ee44ebb5e252bb1ac07ebaed44fa658bfb5f0",
+            "image_path": "percona/percona-postgresql-operator:2.6.0-ppg15.12-pgbouncer1.24.0",
```

Comment:
Hi, what image did we use for testing: percona or perconalab? Hashes differ in them: https://hub.docker.com/layers/perconalab/percona-postgresql-operator/2.6.0-ppg15.12-pgbouncer1.24.0/images/sha256-45a500abd81ea4ab222cac07e2dc714e8631d523a413b20b0846e47dbaa29aaf 

https://hub.docker.com/layers/percona/percona-postgresql-operator/2.6.0-ppg15.12-pgbouncer1.24.0/images/sha256-9ef6204ebf626ee85d2a2afd405ee44ebb5e252bb1ac07ebaed44fa658bfb5f0